exercises/gigasecond/introduction.md

```diff
@@ -0,0 +1,20 @@
+# Introduction
+
+The way we measure time is kind of messy.
+We have 60 seconds in a minute, and 60 minutes in an hour.
+This comes from ancient Babylon, where they used 60 as the basis for their number system.
+We have 24 hours in a day, 7 days in a week, and months?
+Well, for months it depends not only on which month it is, but also what calendar is used in the country you live in.
+
+If we ever colonize Mars or some other planet, it's going to get even messier.
+If someone says "year" do they mean Earth-year or Mars-year?
+
+Instead, we could use the metric system, and count everything in seconds.
+
+- A food recipe might explain that you need to let the brownies cook in the oven for two kiloseconds (that's two thousand seconds).
+- Perhaps you and your family would travel to somewhere exotic for two megaseconds (that's two million seconds).
+- And if you and your spouse were married for _a thousand million_ seconds, you would celebrate your one gigasecond anniversary.
+
+```exercism/note
+The author Vernor Vinge uses the metric system as the basis for time measurements in his science fiction novel, "A Deepness in the Sky".
```

Comment:
Suggest a link to the novel here, for those of us who are die-hard sci-fi fans.  😄  There's [Wikipedia](https://en.wikipedia.org/wiki/A_Deepness_in_the_Sky), but also a review from [TOR publishers](https://www.tor.com/2017/08/03/science-fiction-with-something-for-everyone-a-deepness-in-the-sky-by-vernor-vinge/), that might be "friendlier".